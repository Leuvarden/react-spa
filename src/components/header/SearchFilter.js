import React, { Component } from 'react';
import SearchButton from './SearchButton';

class SearchFilter extends Component {
    
    render () {
        return ( 
            <div className="search-panel__filter">
                <label id="filterDescription">
                    Search by:
                    <button className="search-panel__button-filter">Title</button>
                    <button className="search-panel__button-filter">Genre</button>
                </label>
                <SearchButton />
            </div>
        );
    }
}

export default SearchFilter;