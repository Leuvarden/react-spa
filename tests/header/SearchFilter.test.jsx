import React from 'react';
import SearchFilter from './../../src/components/header/SearchFilter';

/* eslint-disable no-undef */
describe ('SearchFilter', () => {

    it('renders as expected', () => {
        const form = shallow(<SearchFilter />);
    
        expect(form).toMatchSnapshot();
    });

});