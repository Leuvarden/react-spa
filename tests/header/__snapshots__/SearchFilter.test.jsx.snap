// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchFilter renders as expected 1`] = `
<div
  className="search-panel__filter"
>
  <label
    id="filterDescription"
  >
    Search by:
    <button
      className="search-panel__button-filter"
    >
      Title
    </button>
    <button
      className="search-panel__button-filter"
    >
      Genre
    </button>
  </label>
  <SearchButton />
</div>
`;
